getting-started/cloud-multi-node.md

```diff
@@ -0,0 +1,250 @@
+
+Finally, we can create a distributed hypertable and add data to verify that everything is
+set up and working correctly.
+
+```SQL
+-- Create your regular table
+CREATE TABLE sensor_data (
+  time TIMESTAMPTZ NOT NULL,
+  sensor_id INTEGER,
+  temperature DOUBLE PRECISION,
+  cpu DOUBLE PRECISION
+);
+
+-- Convert the table to a distributed hypertable
+SELECT create_distributed_hypertable('sensor_data', 'time', 'sensor_id');
+
+-- Insert some test data to verify you are error free
+INSERT INTO sensor_data VALUES  ('2020-12-09',1,32.2,0.45);
+```
+
+One major point to recognize about the SQL above is the declaration of a partition 
+column (`sensor_id`) for the distributed hypertable. This is intentional, and 
+**recommended**, for distributed hypertable setups. Previously, with regular, 
+single-node hypertables, there was often little benefit in specifying a partition 
+key when creating the hypertable. With distributed hyptertables, however, adding
+a partition key is essential to ensure that data is distributed across data nodes 
+on something more than just time. Otherwise, all data for a specific time range 
+will go to one chunk on one node, rather than being distributed across each data 
+node for the same time range.
+
+## Adding additional database users (optional) [](add-database-users)
+
+One of the other advantages of the user-mapping-based approach is that it allows 
+us to add additional users to the multi-node cluster.
+
+From the **access node**, create a new user and GRANT all privileges. 
+
+```SQL
+CREATE ROLE mn_user1 LOGIN PASSWORD 'password';
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mn_user1;
+```
+
+While connected to the access node we can use a specially designed function to 
+execute SQL commands against data nodes. Notice that the list of nodes is included
+and for this example, it is assumed that both data nodes exist and are added to
+the access node.
+
+```SQL
+CALL distributed_exec(query => 'CREATE ROLE mn_user1 LOGIN PASSWORD $$password$$', node_list => '{dn1,dn2}');
+CALL distributed_exec(query => 'GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mn_user1;', node_list => '{dn1,dn2}');
+```
+
+Finally we add a user mapping for our newly added user so that the AN can connect 
+to the DN with the new user.
+
+```SQL
+CREATE USER MAPPING FOR mn_user1 SERVER dn1 OPTIONS (user 'mn_user1', password 'password');
+```
+
+A few final reminders as you begin to explore the opportunities of a multi-node server:
+
+1. Multi-node clusters can still use _regular_, non-distributed features like 
+regular hypertables, PostgreSQL tables, and continuous aggregations. The data 
+stored in any of these object will reside only on the access node.
+2. There is no limitation on the number of distributed hypertables a user can 
+create on the access node.
+3. Finally, remember that once a Service is marked as an access node or data node,
+ it cannot be used to create another TimescaleDB multi-node cluster.
```

Comment:
```suggestion
 it cannot be used as part of another TimescaleDB multi-node cluster.
```
Question:  Is it that a Service not be used in another cluster, or it can only be part of one cluster at a time?